### ISSUE TITLE
Css not working in production for rails 5

### ISSUE THREAD

**Issue by JalenOng** [mileszs/wicked_pdf#780]
Hi, I ran everything locally and the table is displayed correctly. When I pushed to heroku, the table styling (from bootstrap) is not displayed at all. I only see the contents of the table without the borders of the table which is a styling from bootstrap.

I have included this to the pdf template header:

'''<%= stylesheet_link_tag wicked_pdf_asset_base64('application') %>'''

**Comment by unixmonkey** [comment]
Is `application.css` In your [asset precompile list](https://github.com/mileszs/wicked_pdf/blob/master/README.md#asset-pipeline-usage)?

**Comment by JalenOng** [comment]
Yes, `application.scss` and other stylesheet files are in the asset precompile list
